src/main/java/frc/robot/Robot.java

```diff
@@ -1,25 +1,12 @@
-/*----------------------------------------------------------------------------*/
-/* Copyright (c) 2017-2019 FIRST. All Rights Reserved.                        */
-/* Open Source Software - may be modified and shared by FRC teams. The code   */
-/* must be accompanied by the FIRST BSD license file in the root directory of */
-/* the project.                                                               */
-/*----------------------------------------------------------------------------*/
 package frc.robot;
```

Comment:
I think we want to leave the file header in place.

Reply:
Why? This is our code, even if it originally was generated by their tool. Also, we are violating the terms in that comment, since we don't have any BSD license file.